### PR TITLE
Various little fixes and improvements, as well as beginning writing the TLB documentation

### PR DESCRIPTION
**Pull request by DerekTurtleRoe**
The title 😄 

I basically just did a quick pass on the other documents, and then started writing some TLB docs based on various info from publicly released homebrew documentation. I will most likely be changing this, so that's why it is a draft PR.

Feedback and such would be appreciated! 😄

### CODE REVIEW

File: docs/source/tlb.rst [hunk 1]
```diff
@@ -1,5 +1,43 @@
 TLB
 ===
 
-TLB documentation coming soon
+The TLB (transfer lookaside buffer) converts virtual addresses into physical addresses. The TLB is a fully associative cache with 32 entries, entries are mapped with odd and even numbers in pairs. When a virtual address is given, each entry in the TLB checks the 32 entries for whether they coincide with the address in the "address space identification" (ASID) area in the "Entry Hi" register. If it does, a "hit" occurs, and a physical address is generated in the TLB as well as an offset. If a miss occurs, then an exception occurs and the TLB entry is written by the software to a page table in memory, either to the TLB entry over the selected index register, or it writes to a random entry indicated by the random register. If 2 or more are hits, the TLB isn't correctly executed, and the TLB shutdown (TS) bit of the status register is set to 1, and the TLB cannot be used.
```
Comment (KieronJ) [inline docs/source/tlb.rst:4]: TLB is translation lookaside buffer

Comment (DerekTurtleRoe) [reply docs/source/tlb.rst:4]: Oh, is it? I always have read it was "transfer lookaside buffer"...

I'll update it now. 😄

File: docs/source/boot_process.rst [hunk 2]
```diff
@@ -48,4 +48,4 @@ The first 0x1000 bytes from the cartridge are then copied to SP DMEM. This is im
 
 The program counter is then set to 0xA4000040. Note that this skips the first 0x40 bytes of the ROM, as this is where the header is stored. Also note that execution begins with the CPU executing out of SP DMEM.
 
-The ROM now begins to execute! In practice, this is the Bootcode. A reverse-engineering and analysis of this bootcode can be found `Here <https://www.retroreversing.com/n64bootcode>`_.
+The ROM now begins to execute! In practice, this is the "bootcode". A reverse-engineering and analysis of this bootcode can be found `Here <https://www.retroreversing.com/n64bootcode>`_.
```
Comment (Dillonb) [inline docs/source/boot_process.rst:51]: if you're going to replace this with anything, please replace it with IPL3

File: docs/source/cpu.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ CPU (detailed)
 Registers
 ---------
 
-As noted in the CPU overview, r0 is **hardwired to a value of zero**, and all registers are 64 bits.
+As noted in the CPU overview, r0 is **hardwired to a value of zero**, and all registers are 64-bits.
```
Comment (Dillonb) [inline docs/source/cpu.rst:8]: please remove all these hyphens you added

File: docs/source/interfaces.rst [hunk 4]
```diff
@@ -154,12 +154,12 @@ Video Interface
 -----------------------------------------
 Can be called the VI_STATUS_REG, or the VI_CONTROL REG, whichever you prefer.
 
-This register describes the format of the framebuffer in RDRAM, as well as enables and disables effects such as gamma, dithering, anti-aliasing, etc.
+This register describes the format of the frame buffer in RDRAM, as well as enables and disables effects such as gamma, dithering, anti-aliasing, etc.
```
Comment (Dillonb) [inline docs/source/interfaces.rst:157]: remove the space in the middle of the word "framebuffer" please, here and everywhere else

**Comment by DerekTurtleRoe** [comment]
I think I want to come back to this at a later date, I think right now the information I have is too preliminary. Might revitalize in the future.

**Pull request closed without merging**